community/vncsnapshot/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Ishak BELAHMAR <ishak@nytrio.com>
+# Maintainer: Ishak BELAHMAR <ishak@nytrio.com>
+pkgname=vncsnapshot
+pkgver=1.3
+pkgrel=0
+pkgdesc="The utility allows one to take a snapshot from a VNC server and save it as a JPEG file."
+url="https://github.com/IDNT/vncsnapshot"
+arch="x86_64"
+license="GPL-2.0"
+depends=""
+makedepends="jpeg-dev zlib-dev"
+install=""
+subpackages="$pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/IDNT/vncsnapshot/archive/master.tar.gz"
+builddir="$srcdir"/vncsnapshot-master
+
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+}
```

Comment:
The check function can be omitted. Add before depends, `options="!check" # No test suite available`